lib/susedistribution.pm

```diff
@@ -839,7 +839,7 @@ sub activate_console {
     }
     if (get_var('TUNNELED') && $name !~ /tunnel/) {
         die "Console '$console' activated in TUNNEL mode activated but tunnel(s) are not yet initialized, use the 'tunnel' console and call 'setup_ssh_tunnels' first" unless get_var('_SSH_TUNNELS_INITIALIZED');
-        $self->script_run('ssh -t sut', 0);
+        $self->script_run('ssh -vt sut', 0);
```

Comment:
Won't this interfere with a lot of test cases, where the additional debug output might be unexpected and causes calls like `validate_script_output` ecc to fail in the future?

Reply:
No, one verbose flag won't produce any output when the session is established.
If I'd use it multiple times then the amount of output could start to interfere.

At least that's the conclusion of my research :smile:

Reply:
I agree, if we have connection problems, then we might get something. If we do not have ssh problems during the test, there should not be more messages in the meantime.

If it will be different, we could try using `-E <logfile>` and upload the log

Reply:
The actual output of the ssh tunnel goes via the fifo in `/dev/sshserial`, so the debug output here should not have any functional implications on the running test cases.

Pavel's suggestion is IMHO valid and should not interfere with our test runs. LGTM!

Reply:
We use mostly serial terminal. This will stay in the VNC.